src/test/it/bom.yml

```diff
@@ -0,0 +1,56 @@
+metadata:
+  # labels and annotations are key: value string attributes
+  labels:
+    name: bom-it
+    # Needed to make PCT happy
+    groupId: org.jenkins-ci.main
+    artifactId: jenkins-war
+    version: 2.118-artifact-manager-s3-SNAPSHOT
+spec:
+  core:
+    version: 2.118
+  plugins:
+    - groupId: "org.jenkins-ci.plugins.workflow"
+      artifactId: "workflow-api"
```

Comment:
So this is duplicating information: the `pom.xml` already specifies a dependency on `workflow-api`, and specifies the version. We should not have that twice in the same repo.

I think this is a fundamental flaw in the idea of using a BOM in individual repos for integration tests (as opposed to the overall `jenkins-infra/evergreen/configuration/essentials.yaml`): there is no clear proposal for how to reconcile duplicated configuration. In https://github.com/jenkinsci/jep/pull/92#discussion_r183708147 @carlossg claims that `bom.yml` is the “source of truth” and that `pom.xml` would be updated by “the tooling” but I do not see how this is going to work. Is `pom.xml` not going to be versioned any longer? That seems impractical. So, you would need to change version numbers in both places and commit them at once? What is enforcing that they stay aligned?

To me it seems more comfortable for the POM to specify details of artifacts and their versions, and for the BOM to express _additional_ information, if any—perhaps just delete the `bom.yml` altogether, since the `essentials.yml` here seems to have all the interesting configuration. For purposes of the custom WAR packager, you can just bundle up the `jenkins.war` and Jenkins plugins in Maven’s `test` scope. [This trick](https://github.com/jenkinsci/workflow-aggregator-plugin/blob/2fd92539829c9b066ca4b4b65b50d6c4a403f9e1/demo/Makefile#L5-L12) works fine.

And that would also bypass the “filesystem ref” problem mentioned below.

Reply:
I could create a "POM to BOM" tool which dumps a YAML with all HPI dependencies in the plugin and adds them to BOM.yml (would make sense to rename it to bom-template.yml then).

Reply:
Of course we can do it if needed, in the most of the cases